docs/Configuration.md

```diff
@@ -985,31 +985,30 @@ connect to a network resource is the `url` field.
     ]
 }
 ```
+
 #### GBFS Configuration
 
-Steps to add a GBFS feed to a router:
+[GBFS](https://github.com/NABSA/gbfs) is used for a variety of shared mobility services, with partial support for both v1 and v2.2 ([list of known GBFS feeds](https://github.com/NABSA/gbfs/blob/master/systems.csv)).
 
-- Add one entry in the `updater` field of `router-config.json` in the format
+To add a GBFS feed to the router add one entry in the `updater` field of `router-config.json` in the format:
 
 ```JSON
 // router-config.json
 {
-     "type": "bike-rental",
-     "frequencySec": 60,
-     "sourceType": "gbfs",
-     "url": "http://coast.socialbicycles.com/opendata/"
+   "type": "bike-rental",
+   "sourceType": "gbfs",
+   // frequency in seconds in which the GBFS service will be polled
+   "frequencySec": 60,
+   // The URL of the GBFS feed, under which the well-known GBFS files may be found
+   "url": "http://coast.socialbicycles.com/opendata/"
```

Comment:
Shouldn't we encourage people to use the full URL to the `gbfs.json`?

Reply:
I could reword it to say something along the lines of use `gbfs.json`, and if it doesn't exist than the base url may also be provided. :+1:

Reply:
Great.

Reply:
I've updated the documentation.